templates/Program/program_description.html.twig

```diff
@@ -50,12 +51,24 @@
             {% endif %}
           </span>
       </div>
-      <div id="credits" class="text-break">
-        {% if program.credits %}
-          {{ program.credits|nl2br }}
-        {% else %}
+      {% if program.credits %}
+        <div id="credits" class="text-break">
+            {{ program.credits|nl2br }}
+        </div>
+      {% else %}
+        <div id="credits" class="text-break">
           {{ "noCredits"|trans({}, "catroweb") }}
+        </div>
+      {% endif %}
+      <div id="credits-translation-wrapper" style="display: none;">
+        {% if program.credits %}
```

Comment:
do you need this `if` here? I think if there are no credits to be translated, then you can leave this `div` empty and it will look this same, no? or it has extra space?

Reply:
You are right. I thought if there was an empty div it would cause problems, but after some tests, it looks like it doesn't. I removed the extra if statement